packages/clay-css/src/scss/mixins/_input-groups.scss

```diff
@@ -304,6 +304,20 @@
 				@include clay-input-group-item-variant(
 					map-deep-get($map, focus)
 				);
+
+				&::after {
```

Comment:
We need to move the `&::after` rule-set to line 302. It should get the properties from the `$map` via `map-get($map, after)`. We should have a `&::before` rule-set also. We need to use the clay-css mixin, `@include clay-css(map-get($map, after))`, to print out the properties listed in `after` and `before`. The nested maps in `focus` will print because we are passing it back into the `clay-input-group-item-variant` mixin.

Reply:
Hi @pat270,
I attempted to add these changes, please let me know if this is correct!